masl/test/9288_subtypes/domain.mod

```diff
@@ -0,0 +1,12 @@
+domain subsup is 
```

Comment:
This file should be named subsup.mod rather than domain.mod.  The folder should also have a subsup.int that is simply

domain subsup is
  
end domain;